README.md

```diff
@@ -20,16 +20,12 @@ python setup.py develop
 ```
 
 ## Tests
-To run the tests in your virtualenv, execute:
-```shell
-python setup.py test
-```
-
-Alternatively, you can use `tox` or `detox` to run the tests against multiple versions of python:
+You can use `tox` or `detox` to run the tests against multiple versions of python:
 ```shell
 sudo dnf install python-detox
 detox
 ```
+Test outputs are stored under the `coverage_report` directory as HTML files
```

Comment:
'coverage_report/' you can add link to the md file like https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file